testing/__init__.py

```diff
@@ -0,0 +1,15 @@
+#!/usr/bin/env python
```

Comment:
Should we be setting the environment on files that aren't intended to be main programs?

Reply:
ok, I'll add a rule for handling the shebang line of python files.